robotoff/prediction/upc_image.py

```diff
@@ -0,0 +1,83 @@
+from enum import Enum
+
+import cv2
+import numpy as np
+
+from robotoff.utils import get_logger
+
+logger = get_logger(__name__)
+
+
+class UPCImageType(Enum):
+    """This enum is used to determine the type of image regarding UPC Information"""
+
+    """The image is a UPC Image which means that a UPC has a greater area in the image than the threshold and thus it is a poor quality main photo"""
```

Comment:
You should use comments (`# The image...`) instead of strings here to describe the fields.

Reply:
Done!